src/attrcache.c

```diff
@@ -234,7 +234,6 @@ int git_attr_cache__get(
 		/* remove existing entry */
 		if (file) {
 			attr_cache_remove(cache, file);
-			git_attr_file__free(file); /* offset incref from lookup */
```

Comment:
Might actually be, yeah. I think I came up with similar reasoning once when I touched the threading issues with attr-file caches in the threadmania-tests. But I discarded the complete branch as it was leading nowhere. Might want to revisit this again soonish.